### ISSUE TITLE
Not possible to search for domains.

### ISSUE THREAD

**Issue by MrSoUndso** [2factorauth/twofactorauth#3118]
It currently isn't possible to search for a domain directly:
Example: it is possible to search for Electronic arts, but not for ea.com.
Please change this.

**Comment by kenman345** [comment]
I can take care of this within a few minutes. I actually just added a bunch of data to Jets on Acceptbitcoincash/acceptbitcoincash repository so that we could add stuff to our searching. 

In our case, I made a change to make a hidden field that we threw all the content we wanted to search on instead of only using the 'name' field. If thats okay, I can copy over a lot of that effort.

https://github.com/acceptbitcoincash/acceptbitcoincash/blob/master/_includes/segment/table-row-title.html#L7

We included a bunch of other data components but we can pull back for you to include just the name and the url
